config/initializers/devise.rb

```diff
@@ -252,7 +252,9 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  config.omniauth :cas, host: 'fed.princeton.edu', url: 'https://fed.princeton.edu/cas'
+  config.omniauth(:cas,
```

Comment:
Can we just put these in config.yml?

Reply:
Agreed, I initially attempted this (there was some conflict in the loading procedure), but wanted to get the branch out for testing on Circle first